code/index.html

```diff
@@ -1,37 +1,43 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
   <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
+    <meta charset='UTF-8' />
+    <meta name='viewport' content='width=device-width, initial-scale=1.0' />
+    <meta http-equiv='X-UA-Compatible' content='ie=edge' />
+    <link rel='stylesheet' href='style.css' />
+    <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
     <title>Technigo Project Cluedo</title>
   </head>
   <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
+    <section class='cards'>
+      <h1>Play Cluedo</h1>
+      <article class='killer-deck' id='killerCard'>
         <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
+        <p id='killerName'></p>
+        <p id='killerAge'></p>
+        <p id='killerOccupation'></p>
+        <img id='killerImage' src='' height='75%' />
```

Comment:
We should avoid setting style properties via HTML - all of them should be added in CSS files